bittensor_cli/src/commands/stake/remove.py

```diff
@@ -248,13 +248,13 @@ async def unstake(
             # Additional fields for safe unstaking
             if safe_staking:
                 if subnet_info.is_dynamic:
-                    rate = subnet_info.price.tao or 1
+                    rate = amount_to_unstake_as_balance.rao / received_amount.rao
```

Comment:
I *think* this should be inverted 
```suggestion
                    rate = received_amount.rao / amount_to_unstake_as_balance.rao
```

Reply:
because the price we pass needs to be in TAO